app/styles/templates/organisations/_search_organisations.scss

```diff
@@ -1,7 +1,34 @@
+.organisation_search_container{
```

Comment:
@bharat619 Could you also fix the scoping here ?

I think we can probably move `.search-box-container` and `.search-box` into `.organisation_search_container`. 
`.search-box` is very generic and it can easily conflict with another element somewhere else in the app

I assume we can also put all of `.organisations_search` into `.organisation_search_container`